locales/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-gettext_templates(${CMAKE_CURRENT_SOURCE_DIR} ${ALL_SOURCES})
-gettext_compile(${CMAKE_CURRENT_SOURCE_DIR} share/locale)
-SET_DIRECTORY_PROPERTIES(PROPERTIES CLEAN_NO_CUSTOM TRUE)
-export_var(GETTEXT_ENABLED)
+if (NOT WIN32)
+    gettext_templates(${CMAKE_CURRENT_SOURCE_DIR} ${ALL_SOURCES})
```

Comment:
Oh, I'll take a look at this in more detail. I didn't realize there was a prevention mechanism already
in place. Appveyor suddenly started trying to build the .pot file, breaking all Windows builds.